src/modules/login/index.tsx

```diff
@@ -1,16 +1,60 @@
 import { StackActions, useNavigation } from '@react-navigation/native';
 import * as React from 'react';
-import { View, Text, Button, StyleSheet } from 'react-native';
+import {Text, Image, View, StyleSheet, TextInput, TouchableOpacity} from 'react-native';
 
 import Routes from '../../config/navigation/routes';
 import { useAppDispatch } from '../../hooks/navigation';
 import mockedUsers from '../../mock/users.json';
 import { userActions } from '../profile/redux';
+import {Feather} from "@expo/vector-icons";
+import {LinearGradient} from "expo-linear-gradient";
+
 
 export default function LoginScreen() {
   const navigation = useNavigation();
   const dispatch = useAppDispatch();
   const defaultUser = mockedUsers.results[0];
+  const [data, setData] = React.useState({
+    userName: '',
+    password: '',
+    secureTextEntry: true,
+    isFocusedText: false,
+    isFocusedPassword: false
+  });
+
+  const TextInputChange = (val: string) => {
```

Comment:
Function cannot start with capital letter, it's usually the best to use `camelCase` for this kind of purposes